Pubications.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Publications
+nav: 6
+---
+
+# Publications
+
+G. Salvaneschi, S. Proksch, S. Amann, S. Nadi, M. Mezini, [_On the Positive Effect of Reactive Programming on Software Comprehension: An Empirical Study_](http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7827078), IEEE Transactions on Software Engineering, 2017<br>
+Guido salvaneschi, [_What do we Really Know about Data Flow Languages_](https://dl.acm.org/ft_gateway.cfm?id=3001884&ftid=1805603&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6), PLATEAU 2016<br>
+Guido Salvaneschi, Mira Mezini, [_Debugging Reactive Programming with Reactive Inspector_](https://dl.acm.org/ft_gateway.cfm?id=2893174&ftid=1722250&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6), ICSE 2016<br>
+Guido Salvaneschi, Mira Mezini, [_Debugging for Reactive Programming_](https://saadaziz33.github.io/REScala/2016_Debugging-for-Reactive-Programming_pdf.pdf), ICSE 2016<br>
+Oliver Bracevac, Sebastian Erdweg, Guido Salvaneschi, and Mira Mezini, [_CPL: A Core Language for Cloud Computing_](https://dl.acm.org/ft_gateway.cfm?id=2889452&ftid=1700146&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6), MODULARITY 2016<br>
+[_Reactive Programming: a Walkthrough_](https://dl.acm.org/ft_gateway.cfm?id=2819226&ftid=1617074&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6) G. Salvaneschi, A. Margara, G. Tamburrelli. ICSE Technical Briefing. In Proceedings of the 37th International Conference on Software Engineering (ICSE '15). Florence, Italy. May 16 - 24, 2015.<br>
```

Comment:
remove the <br> tags and use either markdown paragraphs (two newlines) or lists.